frontend/src/pages/customer/Customers.js

```diff
@@ -0,0 +1,198 @@
+import React, {useEffect, useState} from 'react';
+import Table from "react-bootstrap/Table";
+import Form from 'react-bootstrap/Form'
+import {FaEdit} from "react-icons/fa";
+import Page from "../../components/Page";
+import Button from "react-bootstrap/Button";
+import Row from "react-bootstrap/Row";
+import Col from "react-bootstrap/Col";
+import TogglePage from "../../components/TogglePage";
+import ModalAddCustomer from "./ModalAddCustomer";
+import ModalEditCustomer from "./ModalEditCustomer";
+import CardContainer from "../../components/CardContainer";
+import ErrorMessage from "../../messages/errorMessage";
+
+export default () => {
+
+    const [page, setPage] = useState({
+        active: 1,
+        currentPage: 1,
+        countPerPage: 10,
+        countPages: 1
+    });
+    const [customers, setCustomers] = useState([]);
+    const [filter, setFilter] = useState([]);
+    const [lgShow, setLgShow] = useState(false);
+    const [editCustomer, setEditCustomer] = useState({
+        editShow: false,
+        customer: []
+    });
+    const [errorMessage, setErrors] = useState('');
+    const filterOptions={'All': '', 'Active': true, 'Disabled': false};
```

Comment:
Could you please explain why is it not a state of the component? Or what role does it play here in general?